basics/getting_started/update_plugins_format.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Publishing a Plugin to a Custom Plugin Repository
+---
+
+If you intend to use a plugin repository _other than_ the [JetBrains Plugin Repository](https://plugins.jetbrains.com), 
+you will need to:
+* Upload your plugin JAR/ZIP file to the HTTPS web server you are using for the custom repository.
+* Maintain an `updatePlugins.xml` file on that web server describing all the available plugins.
+* Add the URL for the custom repository to the JetBrains IDE [Repository Settings/Preferences](https://www.jetbrains.com/help/idea/managing-plugins.html#repos).
+
+## Describing Your Plugins in an updatePlugins File
+Every custom plugin repository must have at least one `updatePlugins.xml` file to describe the latest compatible version 
+for every hosted plugin. The description in `updatePlugins.xml` is used by JetBrains IDEs to locate plugins by attributes 
+such as id, IDE version, and plugin version. These attributes are displayed by JetBrains IDEs to help users select or upgrade plugins.
+The description also tells the JetBrains IDE where to download the plugin itself.
+
+A custom plugin repository's `updatePlugins.xml` file is constructed and maintained by the administrator of
+the repository. More than one `updatePlugins.xml` file may be required if consumers of the custom repository are using more
+than one version of a JetBrains IDE. For example `updatePlugins-182.xml`, `updatePlugins-183.xml` for IntelliJ IDEA 2018.2 and 2018.3 respectively.
+Each `updatePlugins-*.xml` file will have a unique URL that is added to the JetBrains IDE 
+[Repository Settings/Preferences](https://www.jetbrains.com/help/idea/managing-plugins.html#repos). 
+
+### The Format of an updatePlugins File
+The format of an `updatePlugins.xml` file is simply a list of sequential elements that describe each plugin:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+  The <plugins> element contains the description of the plugins available at this repository. Required. 
+-->
+<plugins>
+  <!-- 
+    Each <plugin> element describes one plugin in the repository. Required.
+    id - used by JetBrains IDEs to uniquely identify a plugin. Required. Must match <id> in plugin.xml
+    url - path to download the plugin JAR/ZIP file. Required. Must be HTTPS
+    version - version of this plugin. Required. Must match <version> in plugin.xml
+  -->
+  <plugin id="fully.qualified.id.of.this.plugin" url="https://www.mycompany.com/my_repository/mypluginname.jar" version="major.minor.update">
+    <!--
+      The <idea-version> element must match the same element in plugin.xml. Required.
+    -->
+    <idea-version since-build="181.3" until-build="184.*" />
```

Comment:
There is no version 184. Branch numbers follow "year/release number" numbering scheme; 183 is followed by 191.